docs/WritingTests.asciidoc

```diff
@@ -1277,8 +1277,25 @@ test module.
 
 ==== Enable snapshots for each module
 
-* Run the worker with --no-cleanup parameter. This will preserve the hard
- disks after test runs.
+* Run the worker with `--no-cleanup` parameter. This will preserve the hard
+ disks after test runs. If the worker(s) are being started via the systemd unit,
+ then this can achieved by pasting the following contents into `systemctl edit
+ openqa-worker@.service` or directly into the file
+ `/etc/systemd/system/openqa-worker@.service.d/override.conf`:
+[source,ini]
+----
+[Service]
+ExecStart=
+ExecStart=/usr/share/openqa/script/worker --no-cleanup --instance %i
+----
+ Aftewards, reload the unit files and restart your workers via:
+[source,sh]
+----
+systemctl daemon-reload
+systemctl restart openqa-worker@1.service
+# optionally restart additional workers, if you have them running
+# systemctl restart openqa-worker@1.service
+----
```

Comment:
There is a much simpler way. The install rules from Makefile create a systemd service "openqa-worker-no-cleanup@.service" that can be used. No need to define your own services, just disable the other services and run the no-cleanup ones instead

Reply:
Didn't know this was a thing… Should be fixed now.